util/chplenv/chpl_3p_libfabric_configs.py

```diff
@@ -0,0 +1,24 @@
+import chpl_locale_model, third_party_utils
+from utils import error, memoize
+
+
+@memoize
+def get_uniq_cfg_path():
+    return third_party_utils.default_uniq_cfg_path()
+
+@memoize
+def get_link_args(libfabric):
+    libs = []
+    if libfabric == 'system':
+      # Try using pkg-config to get the libraries to link
+      # libfabric with.
+      pclibs = third_party_utils.pkgconfig_get_link_args(
+                       'libfabric', system=True)
```

Comment:
This call will use `static=True` by default. Is that what we want? (I'm not sure myself, just checking)

Reply:
For now, yes, because not every conceivable compute node mounts the fallback libfabric library I've installed internally.

Reply:
Gotcha, thanks